eliot-service/eliot/symbolicate_resource.py

```diff
@@ -320,49 +322,96 @@ def get_symcache(self, debug_filename, debug_id, debug_stats):
         end_time = time.perf_counter()
         debug_stats.incr("cache_lookups.time", end_time - start_time)
 
-        # We didn't find it in the cache, so try to download it
```

Comment:
This is one big difference section, but what's going on is that I added some additional timing blocks and I redid the code structure so it returns early when possible with the correct return value. That dedents a lot of this and makes it easier to maintain going forward.